Homework-04-Classes-Files.ipynb

```diff
@@ -21,11 +21,70 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 62,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/path/to/my/file1.txt plaintext 10\n",
+      "path/to/my/file2.txt plaintext 9\n",
+      "path/to/my/file3.txt plaintext 8\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<bound method Text.new_and_measure of <__main__.Text object at 0x11155f4a8>>"
+      ]
+     },
+     "execution_count": 62,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# your code"
+    "class Text:\n",
+    "    def __init__(self, filepath, filetype):\n",
+    "        self.path_to_file=filepath\n",
+    "        self.file_type=filetype\n",
+    "\n",
+    "    def insert_meat(self, text):\n",
+    "        self.file_text=text\n",
+    "        \n",
+    "    def file_size(self, filesize):\n",
+    "        self.file_size=filesize\n",
+    "        #this does not set the size to according to size of object at filepath becasue I do not know what that means\n",
+    "        \n",
+    "    def new_and_measure(self):\n",
+    "        tester=print(input(\"what file would you like to test against?\"))\n",
+    "        if size(tester.file_size) > size(self.file_size):\n",
+    "            print(\"true\")\n",
+    "        \n",
+    "            \n",
+    "t1= Text(\"/path/to/my/file1.txt\", \"plaintext\")\n",
+    "t2= Text(\"path/to/my/file2.txt\", \"plaintext\")\n",
+    "t3= Text(\"path/to/my/file3.txt\", \"plaintext\")\n",
+    "\n",
+    "t1.insert_meat(\"LALALA THIS IS GOING INTO T1\")\n",
+    "t2.insert_meat(\"LALALA THIS IS GOING INTO T2 cool\")\n",
+    "t3.insert_meat(\"LALALA THIS IS GOING INTO T3 WAHOOOOO\")\n",
+    "\n",
+    "t1.file_size(10)\n",
+    "t2.file_size(9)\n",
+    "t3.file_size(8)\n",
+    "\n",
+    "all_texts=[]\n",
+    "\n",
+    "all_texts.append(t1)\n",
+    "all_texts.append(t2)\n",
+    "all_texts.append(t3)\n",
+    "\n",
+    "for obj in all_texts:\n",
+    "    print(obj.path_to_file, obj.file_type, obj.file_size)\n",
+    "    \n",
+    "t1.new_and_measure"
```

Comment:
this refers to the method `new_and_measure` as an object (hence the print out `<bound method Text.new_and_measure of <__main__.Text object at 0x11155f4a8>>`). To call it you need to append `()`.